### PR TITLE
fix(extension): fix to respect ls-path

### PR DESCRIPTION
**Pull request by yjp20**
Currently this extension does not respect the ls-path in the config, this PR should change that. It was mostly copied directly from the vscode version of this plugin.

### CODE REVIEW

**Review by davidroeca — APPROVED** [review]
:+1: LGTM!

@iamcco I'm unclear how to manage releases here so I figure I'll let you merge and manage that process still

**Comment by iamcco** [comment]
@davidroeca What's your npm account, seems I can invite you as maintainer.

**Comment by davidroeca** [comment]
@iamcco https://www.npmjs.com/~davidroeca

**Comment by iamcco** [comment]
> @iamcco https://www.npmjs.com/~davidroeca

Done

**Pull request merged**
